### PR TITLE
python3.pkgs.osmpythontools: 0.2.8 -> 0.2.9

### PR DESCRIPTION
**Pull request by das-g**
###### Motivation for this change

- upstream release 0.2.9 ([GitHub](https://github.com/mocnik-science/osm-python-tools/releases/tag/v0.2.9), [PyPI](https://pypi.org/project/OSMPythonTools/0.2.9/))
- GitHub repo now has tags for releases (as requested at mocnik-science/osm-python-tools#25)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix run nixpkgs.nixpkgs-review
-c nixpkgs-review pr 111847 --post-result`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
  - (there are no executable files, as this is a library)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by das-g** [comment]
Result of `nixpkgs-review pr 111847` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.osmpythontools</li>
    <li>python38Packages.osmpythontools</li>
  </ul>
</details>

File: pkgs/development/python-modules/osmpythontools/default.nix [hunk 1]
```diff
@@ -13,14 +13,17 @@
...
-    inherit version;
-    sha256 = "8a33adbd266127e342d12da755075fae08f398032a6f0909b5e86bef13960a85";
+  src = fetchFromGitHub {
+    owner = "mocnik-science";
+    repo = "osm-python-tools";
+    rev = "v${version}";
+    sha256 = "1qpj03fgn8rmrg9a9vk7bw32k9hdy15g5p2q3a6q52ykpb78jdz5";
   };
 
+  patches = [ ./remove-test-only-dependencies.patch ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/osmpythontools/default.nix:25]: Is there an upsteam patch we can fetch?

Comment (das-g) [reply pkgs/development/python-modules/osmpythontools/default.nix:25]: Not that I know of. And I'm too unsure how test-only dependencies _shoud_ nowadays be declared in Python setuptools-based to file a PR at upstream. (`tests_require` seems to have been deprecated in [setuptools 41.5.0](https://setuptools.readthedocs.io/en/latest/history.html#v41-5-0) (pypa/setuptools#1878) and it's unclear to me what replaces it for setuptools-based projects that do not use tox.)

Comment (SuperSandro2000) [reply pkgs/development/python-modules/osmpythontools/default.nix:25]: Hmmm... Just keep it like this for now.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111847` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.osmpythontools</li>
<li>python38Packages.osmpythontools</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111847` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.osmpythontools</li>
<li>python38Packages.osmpythontools</li>
</ul>
</details>

**Pull request merged**